### DISCUSSION TITLE
$this->dispatch doesn't dispatch

### DISCUSSION THREAD

**Discussion by XternalSoft** [livewire/livewire#6414] [7. Bugs (archived)] [answered]
### Livewire version

v3 beta9

### Which PHP version are you using?

PHP 8.1

### Repository URL

https://github.com/XternalSoft/livewire3-dispatch-bug

### Steps To Reproduce

You can reproduce with my demo repository

1. Create new Laravel project
2. Add Livewire
3. Create two component. 
4. Try to Dispatch...

No error, just no event.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by XternalSoft** [comment]
Oups seems duplicate from https://github.com/livewire/livewire/discussions/6178

**Comment by danie-ramdhani** [comment, marked as answer]
add this on jukebox component:
```php
use Livewire\Attributes\On;
```

**Reply by XternalSoft** [reply]
You're awesome !!! 

I didn't see in the doc....

It's works !

**Reply by danie-ramdhani** [reply]
hahaha sometimes people forget to add that :D
